build-support/bin/release.sh

```diff
@@ -658,6 +677,7 @@ function usage() {
   echo " -l  Lists all pantsbuild packages that this script releases."
   echo " -o  Lists all pantsbuild package owners."
   echo " -e  Check that wheels are prebuilt for this release."
+  echo " -f  Fetch the prebuilt pex for this release and print its path."
```

Comment:
The "release" bit on this line and the one before has gotten less accurate over time: it's any branch/tag SHA.

Reply:
And the one after. Fixed all to just say `for HEAD` which is accurate.